CHANGELOG.md

```diff
@@ -4,9 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Packaging
+
+- Minimum rust version was bumped to `1.41.0`
+
+### Removed
+
+- `create_clipboards` API on Wayland
+- Example for Wayland clipboard
+
 ## 0.6.3
 
-### Added
+### REmove
```

Comment:
```suggestion
### Added
```

Reply:
And now we should realize that we both touching old section.